web/src/styles/app/form.css

```diff
@@ -33,3 +33,33 @@
 .select-condensed {
   height: 2rem;
 }
+
+.rdw-editor-main .DraftEditor-root {
+  min-height: unset; /* override of plugin */
```

Comment:
The plugin CSS specified  a `min-height` of `240px`, and our smallest possible textarea is `100px`, so I'm unsetting the `min-height` to prevent weird gaps.

Reply:
Huh, I've never seen this `unset` value before.  Neat!  Do you know if it works in all browsers/is supported by postcss?  (It looks like the generated CSS still uses `unset` but I'm not sure.

We probably ought to adjust our postcss config to always support IE11, anyway.  Right now if IE11 drops below 2%, our config would stop trying to target it.  😬

Reply:
I went down a rabbit hole with this. [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/min-height#Syntax) lists `unset` as a global value, so I assumed it was kosher. [Caniuse](https://caniuse.com/#feat=css-unset-value), though, specifies that it's not supported in IE11. :-/ So... bummer, and good catch! I'll update it to use `initial` instead.